src/WebJobs.Script.WebHost/WebScriptHostManager.cs

```diff
@@ -149,66 +149,13 @@ public static bool InStandbyMode
             }
         }
 
-        public IReadOnlyDictionary<IHttpRoute, FunctionDescriptor> HttpFunctions
-        {
-            get
-            {
-                return _httpFunctions as IReadOnlyDictionary<IHttpRoute, FunctionDescriptor>;
-            }
-        }
-
-        public async Task<HttpResponseMessage> HandleRequestAsync(FunctionDescriptor function, HttpRequestMessage request, CancellationToken cancellationToken)
-        {
-            // ensure that the host is ready to process requests
-            await DelayUntilHostReady(_hostTimeoutSeconds, _hostRunningPollIntervalMilliseconds);
```

Comment:
Asked above about where HttpRequestManager throtting logic went. This delay logic and this entire function were used in conjunction with that.